CMakeLists.txt

```diff
@@ -303,9 +303,6 @@ function(set_gnu_base_compile_options target)
     target_compile_options(${target} PRIVATE $<$<CONFIG:Check>:-Os>)
     target_compile_options(${target} PRIVATE $<$<CONFIG:CheckFull>:-Os -Wcast-qual>)
 
-    if(TF_PSA_CRYPTO_FATAL_WARNINGS)
-        target_compile_options(${target} PRIVATE -Werror)
```

Comment:
As in https://github.com/Mbed-TLS/mbedtls/pull/10271#pullrequestreview-3011185536, this is too broad.